packages/react/src/components/FileUploader/Filename.tsx

```diff
@@ -11,11 +11,12 @@ import React from 'react';
 import Loading from '../Loading';
 import { usePrefix } from '../../internal/usePrefix';
 import { ReactAttr } from '../../types/common';
-
 export type FilenameStatus = 'edit' | 'complete' | 'uploading';
 
+type SVGAttr = React.SVGAttributes<React.ReactSVGElement>;
+
 export interface FilenameProps
-  extends Omit<ReactAttr<HTMLElement>, 'tabIndex'> {
+  extends Omit<ReactAttr & SVGAttr, 'tabIndex' | 'type'> {
```

Comment:
I'm not super familiar with this code, but I'm curious as to why this type change was needed as a part of the Icons type work.

Reply:
@jdharvey-ibm it's because this switch: https://github.com/carbon-design-system/carbon/blob/main/packages/react/src/components/FileUploader/Filename.tsx#L59 makes the Filename component return either `button` or a `CheckmarkFilled` icon component. All icons inherit SVG attributes.

Reply:
Ahh okay awesome. Thanks for clarifying!